app/notifications/notifications-list/notifications-list.component.ts

```diff
@@ -13,12 +13,21 @@ import { NotificationService } from '../notification.service';
 })
 export class NotificationsListComponent {
   @Input() notifications: Notification[];
+  @Input() selectedStatus: string = '';
+  @Input() selectedNotificationType: string = '';
+  getFilterSortingOptions: boolean = false;
```

Comment:
please rename this variable. get... sounds for me like a getter function. 
something starting with is... is a good choice for a boolean (we should put that in the CG as well maybe :D )